optimizely/project_config.py

```diff
@@ -215,6 +195,28 @@ def get_rollout_experiments(self, rollout):
 
         return rollout_experiments
 
+    def get_rules_for_flag(self, feature_flag):
+        rules = map(lambda exp_id: self.experiment_id_map[exp_id], feature_flag['experimentIds'])
+        rules = list(rules)
+        rollout = None if len(feature_flag['rolloutId']) == 0 else self.rollout_id_map[feature_flag['rolloutId']]
+
+        if rollout:
+            rollout_experiments = rollout.experiments
+            for exp in rollout_experiments:
+                rules.append(self.experiment_id_map[exp['id']])
```

Comment:
I understand experiment_id_map is for ab-tests only, not including rollout rules. Did it pass our tests?

Reply:
All unit tests and FSC is passing.